docs/rfcs/managed.md

```diff
@@ -0,0 +1,150 @@
+
+### Give a comprehensive, out-of the box experience of OpenTelemetry for a first-time user
+The goal is to delight first-time customers by giving them a complete view of what is possible with OpenTelemetry.
+
+### Do not offer a way to turn on and off specific features
+We mean to offer a specific functional coverage of a specific use case that offers best practices of OpenTelemetry.
+
+We don't allow to toggle features in and out specifically because we want to avoid increasing the need for functional test coverage.
+
+If customers want to increase their level of control of the feature set, they can remove the managed custom resource and instead
+follow the traditional route of setting up collectors, target allocators, instrumentation custom resources as is possible today.
+
+### OTLP, single endpoint support
+
+The solution exports data to a single endpoint using the OTLP protocol. Customers may choose to deploy a gateway
+to transform, filter, redact data before sending it to storage.
+
+### Context-aware
+
+Best practices differ between platforms, cloud providers, available endpoints, node operating system and so on.
+
+The solution will enable features based on those parameters. For example, it will deploy Openshift-specific features when deployed on the Openshift Cloud Platform.
+
+## Use cases for proposal
+
+### Initial installation
+
+The customer wants to install the operator and immediately get value from the installation.
+
+Upon installation, they install the custom resource with information on an OTLP endpoint where they can direct all data.
+
+### Upgrade
+
+The customer wants to upgrade their operator. All assets are upgraded out of the box in concert as part of the upgrade,
+and are guaranteed to work through functional test coverage.
+
+### Deletion and reinstallation
+
+The customer can cleanly delete the custom resource, which triggers the deletion of all OpenTelemetry assets under
+management by the operator. No stragglers are left.
+
+### Infrastructure monitoring
+
+The customer upon installation of the custom resource can see Kubernetes cluster metrics, and kubeletstats metrics.
+
+The customer can follow events and receive Kubernetes entity data.
+
+The customer upon installation receives metrics from hostmetrics receiver with its default configuration.  Note the metrics map to the host, meaning the collector will have access to host volumes to scrape their utilization.
+
+### Workload monitoring
+
+All standard CRDs for instrumentations are all enabled by default for all namespaces.
+
+### Node logs
+
+The customer upon installation will receive logs from the node operating system, as well as pod logs from Kubernetes, scraped with the filelog receiver from the file system.
+
+## Configuration
+
+The customer can enable exporting signals individually: metrics, logs, traces and profiles.
+
+## Struct Design
+
+```go
+import "go.opentelemetry.io/collector/exporter/otlpexporter"
+
+type ManagedCustomResource struct {
```

Comment:
I don't love the name Managed here because we have that as a field on the collector already. I would prefer something like `MonitoringStack` or `ClusterMonitor` or `ClusterObservability`. Something that makes it clear that this resource is deploying a way to monitor the entirety of the user's cluster.

Reply:
Naming is hard. I thought it over, `ClusterObservability` I guess works